src/rewrites/rewrite_www.conf

```diff
@@ -22,7 +22,6 @@
 
 <IfModule mod_rewrite.c>
     RewriteEngine On
-    RewriteCond %{HTTPS} !=on
```

Comment:
@xtratio Could you add a comment in the heading part as @ManasN (https://github.com/h5bp/server-configs-apache/pull/84#issuecomment-289069219)?